content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
 | Finish          |       3 |       4 | FT, FTN, FTV, F2V    |
 | Situation       |     2/0 |     4/0 | SIT                  |
 
+Question Type Minimums
+: For a given Question Type, the minimum number of questions must be asked in Questions Numbered 1-20. This excludes any questions appended with "A" or "B."
+
+Question Type Maximums
+: For a given Question Type, the maximum number of questions must not be exceeded in all "non-Overtime" questions. This includes any questions appended with "A" or "B."
+
+Overtime Specifics
+Question Type maximums can be exceeded in Overtime.
+
+## Question Writing
+
+### Preparation of Questions
+All questions must contain verbatim material. On some types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+
+### Question Types
+
+#### Interrogative
+Should not be overly long and should be clear and precise.
+Must have exactly one inserted interrogative word.
+Must have a single answer.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+The Question Type abbreviation is "INT"
+
+#### Multiple Answer
+Must have exactly one inserted interrogative word.
+Must have more than one answer.
+- The clarification of a single answer does not meet the above requirement.
+- The identification of a singular pronoun does not meet the above requirement.
+- The identification of a plural pronoun does meet the above requirement.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+If the multiple answers contain at least one "positive" answer AND at least one "negative" answer, this is deemed invalid as a Multiple Answer question.
+The Question Type abbreviation is "MA"
+
+#### Reference
+Used to distinguish exact same words or phrases from the material.
+The entire question is part of the required question + answer.
+If a quizzer needs to provide the reference question, the provided reference question must: 
+- Be of the exact same type as the one on the card (CR, CVR, CRMA, CVRMA).  The quizmaster and/or answer judge must check to confirm this.
+- Have content AND wording is similar to the content and wording of the written question.
+    - For greater clarity, we have provided the following example: CVR According to Luke chapter 23: 37 King of what?  A. The Jews The quizzer can give the question “King of what?” or “King of whom?”, as Jews could be referring to either nation (what) or people (whom).  In this case the quizzer could also give “King of the what?”
+- Have the determining word or phrase from the quizmaster’s card. 
+    - In the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar.
+The determination of a valid Reference question, and the valid sub-type only considers the content in the question. 
+
+##### Chapter-Verse
+Must have a single answer.
+The question must contain a word or phrase that exists more than once in the same chapter.
```

Comment:
I did not include language about a word/phrase that exists more than once in a chapter, but in the same verse, which could potentially mean a unique word can be in a valid reference question. Thoughts?

Reply:
The original rulebook has issues here. For functional parity, we shouldn't change this right now.

Reply:
thoughts @gryphonshafer ?  While a small change in application, I think it's a very big change in substance. I totally agree with the change BTW.

Reply:
I agree. "Func equiv" means the rulebook we end up with represents what IBQ actually does, not necessarily what the current rulebook says. We should note these sort of changes, so the CQLT/DCs can be pointed to them. But we can easily defend this and others as necessary to keep func equiv.

Reply:
Hmmm, I don't disagree. But I think the four of us should absolutely agree that "functional equivalence" is about what "IBQ folks generally do > what the rule book currently literally says"

Reply:
Agreed. Everybody should agree with me. :) In all seriousness, I'm seeing that I should write up a "this is what func equiv" is document in the root folder, and I should also author a style conventions section, maybe to exist in the "index.md" for the rulebook, or maybe root folder.